### PR TITLE
Changes from ONNXRuntime branch

### PR DESCRIPTION
**Pull request by orausch**

### CODE REVIEW

**Review by tbennun — APPROVED** [review]

File: dace/sdfg/validation.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def validate_state(state: 'dace.sdfg.SDFGState',
             raise
         except Exception as ex:
             raise InvalidSDFGNodeError("Node validation failed: " + str(ex),
-                                       sdfg, state_id, nid)
+                                       sdfg, state_id, nid) from ex
```
Comment (tbennun) [inline dace/sdfg/validation.py:158]: @orausch why is this here?

**Review by tbennun — CHANGES_REQUESTED** [review]

File: dace/sdfg/validation.py (hunk 1 above)
Comment (orausch) [reply dace/sdfg/validation.py:158]: This was helpful for me while debugging so I thought it could be useful for master. I can remove it if you disagree

**Review by tbennun — APPROVED** [review]
